.github/workflows/automatic_code_review.yml

```
@@ -0,0 +1,81 @@
+name: Automatic code review
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  review_code:
+    name: Run Danger 
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Checkout source code
+        uses: actions/checkout@v2.3.2
+
+      - name: Load Gradle cached
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches/modules-*
+            ~/.gradle/caches/jars-*
+            ~/.gradle/caches/build-cache-*
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      # Run Detekt and generate Detekt reports
+      - name: Run Detekt on CoroutineTemplate
+        working-directory: ./CoroutineTemplate
+        run: ./gradlew detekt
+
+      - name: Run Detekt on RxJavaTemplate
+        working-directory: ./RxJavaTemplate
+        run: ./gradlew detekt
+
+      # Run Android Lint and generate Lint reports
+      - name: Run Android Lint on RxJavaTemplate
+        working-directory: ./RxJavaTemplate      
+        run: ./gradlew lint
+
+      - name: Run Android Lint on CoroutineTemplate
+        working-directory: ./CoroutineTemplate      
+        run: ./gradlew lint
+
+        # Run Unit Test & Jacoco for code coverage for RxJavaTemplate
```

wrong or inconsistent indentation 😄 
```suggestion
       # Run Unit Test & Jacoco for code coverage for RxJavaTemplate
```